### PR TITLE
Feature/carousel

### PR DESCRIPTION
**Pull request by tantany**

### CODE REVIEW

File: components/Carousel.js [hunk 1]
```diff
@@ -21,47 +21,44 @@ class Carousel {
   }
 
   _rotate(direction) {
-    const carouselTrack = this._carouselElement.querySelector('.carousel__track');
-    carouselTrack.style.transition = "transform 0.6s ease-in-out";
+    this._carouselTrack.style.transition = "transform 0.6s ease-in-out";
     this._counter += direction;
-    carouselTrack.style.transform = 'translateX(' + ( -this._sizeOfSlide * this._numberOfSlidesPerPage * this._counter ) + 'px)';
+    this._carouselTrack.style.transform = 'translateX(' + ( -this._sizeOfSlide * this._numberOfSlidesPerPage * this._counter ) + 'px)';
```
Comment (julieg18) [inline components/Carousel.js:26]: Would this look cleaner if it was made with `` instead of ''?

File: components/Carousel.js [hunk 2]
```diff
@@ -21,47 +21,44 @@ class Carousel {
...
     this._changeIndicator(direction);
   }
 
   _restart() {
-    const carouselTrack = this._carouselElement.querySelector('.carousel__track');
-    carouselTrack.style.transition = "transform 0.6s ease-in-out";
-    carouselTrack.style.transform = 'translateX(' + 0 + 'px)';
+    this._carouselTrack.style.transition = "transform 0.6s ease-in-out";
+    this._carouselTrack.style.transform = 'translateX(' + 0 + 'px)';
```
Comment (julieg18) [inline components/Carousel.js:32]: I believe you can just write this as `'translateX(0px)';`.

File: components/Carousel.js [hunk 3]
```diff
@@ -76,46 +73,46 @@ class Carousel {
...
   }
 
   _setEventListeners() {
-    const carouselTrack = this._carouselElement.querySelector('.carousel__track');
     const prevButton = this._carouselElement.querySelector('.carousel__slide-button_type_prev');
     const nextButton = this._carouselElement.querySelector('.carousel__slide-button_type_next');
     //Button Listeners
     nextButton.addEventListener('click', () => {
-      carouselTrack.style.animationPlayState = "paused";
+      this._carouselTrack.style.animationPlayState = "paused";
```
Comment (julieg18) [inline components/Carousel.js:91]: These should be single quotes for consistency.

File: pages/index.js [hunk 4]
```diff
@@ -1,7 +1,8 @@
-import { Carousel } from "./Carousel.js";
-
-const carouselList = document.querySelectorAll(".carousel");
-const donateBtn = document.querySelector(".donations__submit-button");
+import { Carousel } from "../components/Carousel.js";
```
Comment (julieg18) [inline pages/index.js:1]: These also should be single quotes.

File: utils/constants.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+const carouselList = document.querySelectorAll(".carousel");
+const donateBtn = document.querySelector(".donations__submit-button");
```
Comment (julieg18) [inline utils/constants.js:2]: These also should be single quotes.

File: components/Carousel.js [hunk 6]
```diff
@@ -21,47 +21,44 @@ class Carousel {
...
-  //indicators are the dots in the buttom of the carousel
+  // indicators are the dots in the buttom of the carousel
   _setIndicators() {
-    const carouselTrack = this._carouselElement.querySelector('.carousel__track');
-    const carouselIndicators = this._carouselElement.querySelector('.carousel__indicators');
+    const carouselIndicatorContainer = this._carouselElement.querySelector('.carousel__indicator-container');
 
     const button = [];
     for(let i=0; i < this._numberOfTransitions + 1; i++){
       button[i] = document.createElement("BUTTON");
```
Comment (julieg18) [inline components/Carousel.js:43]: These should be single quotes.

File: components/Carousel.js (hunk 1 above)
Comment (tantany) [reply components/Carousel.js:26]: What do you mean?

Comment (julieg18) [reply components/Carousel.js:26]: Normally when you're placing variables inside of strings you use backticks. And place your variables inside with ${}

File: components/Carousel.js (hunk 2 above)
Comment (tantany) [reply components/Carousel.js:32]: done

File: components/Carousel.js (hunk 3 above)
Comment (tantany) [reply components/Carousel.js:91]: done

File: pages/index.js (hunk 4 above)
Comment (tantany) [reply pages/index.js:1]: done

File: utils/constants.js (hunk 5 above)
Comment (tantany) [reply utils/constants.js:2]: done

**Pull request merged**